src/commands/coverage/README.md

```diff
@@ -68,4 +68,5 @@ Will upload ode coverage report file src/commands/coverage/__tests__/fixtures/ja
 
 ## Further reading
 
-[//]: <> (TODO: Add link to the documentation page)
```

Comment:
This line is invisible for now. Let's not change that.
https://www.markdownguide.org/basic-syntax/#reference-style-links